### ISSUE TITLE
Do not fully hide numbers when "highlight unsolved numbers" is enabled

### ISSUE THREAD

**Issue by istudyatuni** [lucasnlm/antimine-android#370]
**Is your feature request related to a problem? Please describe.**

My suggestion - do not hide numbers whose cells are not completely solved, because I use "allow tap on numbers", and when around a cell with a number all mines are found, the number is hidden regardless of the presence of unopened cells around

**Describe the solution you'd like**

Hide numbers only when all cells around are either marked as mine or opened

**Describe alternatives you've considered**

**Additional context**

**Comment by istudyatuni** [comment]
But I'm not sure, I think it wouldn't be good to paint it in the same color as an unsolved cell.

**Comment by lucasnlm** [comment]
Currently it's doesn't hide the numbers, it actually make them translucent.
On your phone the numbers are not visible? If so, I can increase the transparency or dim them some other way (like you said).

**Comment by istudyatuni** [comment]
No, sorry, by "hidden" I mean that they are low-contrast, almost invisible

<details>

![image](https://user-images.githubusercontent.com/43654815/160987287-e235b7c8-6696-4543-b0a3-52eaff60e1e8.png)

</details>

**Comment by istudyatuni** [comment]
>  I can increase the transparency or dim them some other way

That would be great!
